lighthouse-core/test/audits/critical-request-chains-test.js

```diff
@@ -100,6 +105,7 @@ describe('Performance: critical-request-chains audit', () => {
 
   it('calculates the correct chain result for a real trace', () => {
     const artifacts = {
+      traces: {defaultPass: createTestTrace({topLevelTasks: [{ts: 0}]})},
```

Comment:
can we not use a real trace here? should change the test name if not, but add another so we still have at least one complete test using real artifacts

Reply:
I would, if there was one checked in. I can never remember how to make one. And would I also need to update the devtools log file?

Does a real trace even matter for the graph here?

Reply:
we should have several pairs checked in, the https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/test/fixtures/traces/progressive-app-m60.json, LCP pair, etc

> Does a real trace even matter for the graph here?

It does for the creation of page graph relationships, so ensuring we maintain at least one real world test for this audit seems reasonable.

Reply:
Oh and also to make sure the code can actually handle CPU nodes correctly. I don't think anything here has a single CPU node, so if we weren't checking `node.type === 'network'` for example it might have bugs.